roles/install_nextcloud/tasks/tls_signed.yml

```diff
@@ -29,28 +29,30 @@
     dest: "{{ nextcloud_tls_cert_file }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: "0640"
+    mode: u=rw,g=r,o=-
   vars:
     input_files: ["{{ nextcloud_tls_src_cert }}", "{{ nextcloud_tls_src_chain }}"]
   when:
     - nextcloud_tls_src_chain is defined
     - nextcloud_websrv in ["nginx"]
 
-- name: "[SIGNED TLS] - Key is copied to the host"
+- name: "[SIGNED TLS] - Copy certificate chain file for apache2 to the host"
   ansible.builtin.copy:
-    dest: "{{ nextcloud_tls_cert_key_file }}"
-    src: "{{ nextcloud_tls_src_cert_key }}"
+    dest: "{{ nextcloud_tls_chain_file }}"
+    src: "{{ nextcloud_tls_src_chain }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: "0400"
+    mode: u=rw,g=r,o=-
     force: false
+  when:
+    - nextcloud_tls_src_chain is defined
+    - nextcloud_websrv not in ["nginx"]
 
-- name: "[SIGNED TLS] - Certificate chain is copied to the host"
+- name: "[SIGNED TLS] - Key is copied to the host"
   ansible.builtin.copy:
-    dest: "{{ nextcloud_tls_chain_file }}"
-    src: "{{ nextcloud_tls_src_chain }}"
+    dest: "{{ nextcloud_tls_cert_key_file }}"
+    src: "{{ nextcloud_tls_src_cert_key }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: "0640"
+    mode: u=r,g=-,o=-
```

Comment:
Have you changed it from 0400 to 0640 on purpose here?

Reply:
Actuallly the change is from 0640 to 0400. Here 0640 is appropriate, this is a certificate, not a key. So seems I mixed this with the former permissions up ... . As before, is it better your propose the change to the mode or should I do this change?